.github/workflows/release.yml

```diff
@@ -0,0 +1,70 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+      - next
```

Comment:
@rafegoldberg I don't think we want to build releases against `next` here because of the amount of Dependabot updates that we get so it's probably safe to remove this and just build a new release whenever `next` is merged into `master`, right?

Reply:
Yeah, that's what we do on ReadMe as well.

Reply:
We should also [update the release branches](https://github.com/readmeio/api-explorer/pull/828#discussion_r453998516).